operators/awx-operator/1.2.0/metadata/annotations.yaml-e

```diff
@@ -0,0 +1,14 @@
+annotations:
```

Comment:
@john-westcott-iv I am not entirely sure how this happened, but this file seems to have been added by mistake.  It is not in previous PR's and seems to just be a copy of `operators/awx-operator/1.2.0/metadata/annotations.yaml`

Could you please remove it?